agent/php_user_instrument.c

```diff
@@ -62,7 +62,24 @@ int nr_zend_call_orig_execute(NR_EXECUTE_PROTO TSRMLS_DC) {
   zend_end_try();
   return zcaught;
 }
-
+#if ZEND_MODULE_API_NO >= ZEND_8_0_X_API_NO /* PHP8+ */
+int nr_zend_call_oapi_special_before(nruserfn_t* wraprec,
+                                     nr_segment_t* segment,
+                                     NR_EXECUTE_PROTO TSRMLS_DC) {
```

Comment:
`TSRMLS_*` no longer necessary in PHP7+
```suggestion
                                     NR_EXECUTE_PROTO) {
```